src/index.jsx

```diff
@@ -146,6 +147,14 @@ var MaskedInput = React.createClass({
       setTimeout(this._updateInputSelection, 0)
       this.props.onChange(e)
     }
+    else {
```

Comment:
The falsy return happens here https://github.com/insin/inputmask-core/blob/master/lib/index.js#L374-L393 on line 391. I'm not exactly sure what that means, but I can dive into it more if need be.